src/kvstore/NebulaStore.cpp

```diff
@@ -387,27 +387,31 @@ void NebulaStore::asyncRemovePrefix(GraphSpaceID spaceId,
 
 
 ResultCode NebulaStore::ingest(GraphSpaceID spaceId,
-                               const std::string& extra,
-                               const std::vector<std::string>& files) {
+                               const std::string& extra) {
     decltype(spaces_)::iterator it;
     folly::RWSpinLock::ReadHolder rh(&lock_);
     RETURN_IF_SPACE_NOT_FOUND(spaceId, it);
     for (auto& engine : it->second->engines_) {
         auto parts = engine->allParts();
         std::vector<std::string> extras;
         for (auto part : parts) {
+            LOG(INFO) << "Loading Part " << part;
+            auto path = folly::stringPrintf("%s/%d", extra.c_str(), part);
+            auto files = nebula::fs::FileUtils::listAllFilesInDir(path.c_str(), true, "*.sst");
+
+            if (files.size() == 0) {
```

Comment:
Do we always have sst files in a partition? Maybe some partitions have, some don't, WDYT.

Reply:
Good catch  I should check path before list files.